packages/roosterjs-editor-plugins/lib/pluginUtils/DragAndDropHelper.ts

```diff
@@ -24,29 +35,50 @@ export default class DragAndDropHelper<TContext, TInitValue> implements Disposab
         private context: TContext,
         private onSubmit: (context: TContext, trigger: HTMLElement) => void,
         private handler: DragAndDropHandler<TContext, TInitValue>,
-        private zoomScale: number
+        private zoomScale: number,
+        isMobile: boolean = false
```

Comment:
same here, we can use Browser.ts to detect if it is in mobile, and no need to manually pass it in.

Reply:
Refactored it as you suggested